www/js/app.js

```diff
@@ -768,10 +833,20 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                 $('#articleListHeaderMessage').empty();
                 $('#articleListWithHeader').hide();
                 $("#prefix").val("");
-                iframeArticleContent.onload = function () {
+                iframeArticleContent.onload = function() {
                     // The content is fully loaded by the browser : we can hide the spinner
-                    iframeArticleContent.onload = function () {};
                     $("#searchingArticles").hide();
+                    // Deflect drag-and-drop of ZIM file on the iframe to Config
+                    var doc = iframeArticleContent.contentDocument.documentElement;
+                    var docBody = doc ? doc.getElementsByTagName('body') : null;
+                    docBody = docBody ? docBody[0] : null;
```

Comment:
@mossroy Is this enough checking of intermediate variables? I guess I could check that the iframe has a `contentDocument` before checking for the `documentElement`, but elsewhere we just do `iframeArticleContent.contentDocument.documentElement`, so it seems unnecessary.

I use the same code in line 994 below.

Reply:
Yes, I think it's enough. I wanted to cover at least the case where the iframe would be empty